<?xml version="1.0"?>
<launch>
    <!--Launch Zed Wrapper-->
	<include file="$(find zed_wrapper)/launch/zed0.launch"/>

    <!--Down sampling-->
    <param name="camera/ds_size" type="double" value="0.05" />
    <node pkg="sensor_fusion" type="camera_downsample" name="camera_downsample">
        <remap from="/cloud" to="/zed0/point_cloud/cloud_registered" />
        <remap from="/output" to="/zed0/ds_cloud" />
    </node>

    <node pkg="sensor_fusion" type="optical_flow.py" name="optical_flow_zed0" >
        <remap from="image" to="/zed0/left/image_rect_color/republish" />
        <remap from="optical_flow" to="/zed0/optical_flow" />
        <remap from="mode" to="/zed0/move" />
    </node>
</launch>
